android/cli/commands/_build.js

```diff
@@ -2817,91 +2819,51 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 		if (!jsFilesToEncrypt.length) {
 			// nothing to encrypt, continue
 			return next();
-		}
-
-		// figure out which titanium prep to run
-		let titaniumPrep = 'titanium_prep';
-		if (process.platform === 'darwin') {
-			titaniumPrep += '.macos';
-		} else if (process.platform === 'win32') {
-			titaniumPrep += '.win32.exe';
-		} else if (process.platform === 'linux') {
-			titaniumPrep += '.linux' + (process.arch === 'x64' ? '64' : '32');
-		}
+		} else if (cloak) {
+			this.logger.info('Encrypting javascript assets...');
+			appc.async.parallel(this,
```

Comment:
I know we haven't started to use async/await yet in our CLI code but there's no reason *not* to at this point. We're Node 8 minimum and about to move to 10 as min in SDK 9.

I think this code would be much more readable if we moved to calling an async function rather than nesting these callbacks.

I know our ESLint rules don't like mixing callbacks and promises, but we kind of need to to start transitioning over.